### ISSUE TITLE
Korean initials improperly formatted

### ISSUE THREAD

**Issue by acrellin** [skyportal/skyportal#1567]
Breaking this out from another issue. 

@AlexGKim  can you please provide more details here, e.g. the exact text you input as first, last name and what you expect to see in the profile avatar? Thanks

**Comment by AlexGKim** [comment]
I suggest using the first name only, not the first characters of the first and last names only.  In most all cases the first name will be <=2 characters.

**Comment by kmshin1397** [comment]
Agreed - just the full first name would make sense for Korean names.

**Comment by acrellin** [comment]
I'm looking into language detection libraries

**Comment by kmshin1397** [comment]
@acrellin I also tried looking around but I found that names are not generally handled well because the string lengths were too short (the detection libraries out there are generally based on models I think so it's hard for them to really determine a language off of a few characters without distinctive words, etc.

For this specific issue, it's pretty easy to just regex match for Korean alphabet characters as no other language uses them. I can make a PR for it soon.

If you're curious, this is the best library I found: https://github.com/wooorm/franc
